pkg/controllers/resources/services/translate.go

```diff
@@ -124,6 +119,12 @@ func (s *serviceSyncer) translateUpdate(pObj, vObj *corev1.Service) *corev1.Serv
 		updated.Spec.SessionAffinityConfig = vObj.Spec.SessionAffinityConfig
 	}
 
+	// load balancer source ranges
+	if !equality.Semantic.DeepEqual(vObj.Spec.LoadBalancerSourceRanges, pObj.Spec.LoadBalancerSourceRanges) {
+		updated = newIfNil(updated, vObj)
+		updated.Spec.LoadBalancerSourceRanges = pObj.Spec.LoadBalancerSourceRanges
```

Comment:
This should use vObj on the right side, shouldn't it? 
`updated.Spec.LoadBalancerSourceRanges = vObj.Spec.LoadBalancerSourceRanges`